README.md

```diff
@@ -1,19 +1,44 @@
 # toffy
-Scripts for interacting with and generating data from the commercial MIBIScope
+Scripts for interacting with and generating data from the commercial MIBIScope.
+
+### System
+Depending on what operating system you are planning to install toffy on, the requirements will differ slightly.
+- [Install on Windows](#windows)
+- [Install on macOS](#macos)
 
 ## Requirements
+### Windows
 
-Latest version of conda (miniconda prefered)
+- You must have **C++ Build Tools** (VS19) installed. 
+Go to  https://visualstudio.microsoft.com/visual-cpp-build-tools/ and click 'Download Build Tools'.
+Open the installer and then follow the prompts.
```

Comment:
we should note which pack/bundle they need to install, since 'build tools' is a gui for installing the actual build tools (very helpful, microsoft)

For reference, it looks like it can be named any of the following:
 - Visual C++ build tools
 - Desktop development with C++
 - C++ buildtools